frontend/src/pages/Grading/index.tsx

```diff
@@ -33,9 +35,33 @@ const GradingAssignmentRow: React.FC<IGradingAssignmentRow> = ({
   );
   const navigate = useNavigate();
 
+  const downloadGrades = () => {
+    const csvContent = studentAssignments
+      .map((work) =>
+        work.contributors
+          .map((contributor) => {
+            return `${contributor},${work.auto_grader_score},${work.manual_feedback_score},${!work.auto_grader_score && !work.manual_feedback_score ? null : (work.auto_grader_score ?? 0) + (work.manual_feedback_score ?? 0)}`;
```

Comment:
Let's extract the last column value (`!work.auto_grader_score && !work.manual_feedback_score ? null : (work.auto_grader_score ?? 0) + (work.manual_feedback_score ?? 0)`) into a defined variable beforehand for readability. 

Also, since the way our final score is computed may change or be dynamic depending on professors' grading schemes, you alternatively could put this logic into a helper function.